native/cmake/predefine.cmake

```diff
@@ -132,6 +139,10 @@ else()
     message(STATUS "Ignore NO_WERROR")
 endif()
 
+if("${OHOS_ARCH}" STREQUAL "armeabi-v7a")
```

Comment:
OHOS_ARCH does not exist on the other platform.

Reply:
Yes, so other platforms will not go here, and the use of openharmony's macros also requires judgment

Reply:
I think using platform macro to wrap the specific platform code is more explicit.
```cmake
if (OPENHARMONY)
  if ($ARCH STREQUAL "armeabi-v7a")
    ...
  endif()
endif()
```

Reply:
I think using platform macro to wrap the specific platform code is more explicit.
```cmake
if (OPENHARMONY)
  if ($ARCH STREQUAL "armeabi-v7a")
    ...
  endif()
endif()
```